functions/__fzf_extract_var_info.fish

```diff
@@ -6,17 +6,17 @@ function __fzf_extract_var_info --argument-names variable_name set_show_output -
     # $variable_name[
     string match --entire --regex "^\\\$$variable_name(?:: set|\[)" <$set_show_output |
```

Comment:
Self-note for later: why are 3 `\`s necessary to escape the first `$`? Well, the last one is needed to prevent fish from starting a variable substitution. Then, to prevent the regex library from interpreting the `$` as an anchor, we need to prefix it with a `\`. But to prevent that `\` from applying to the now-literal `$`, we need to escape it as well with another `\`.